forms-flow-components/src/components/CustomComponents/FormBuilderModal.tsx

```diff
@@ -120,7 +126,21 @@ export const FormBuilderModal: React.FC<BuildFormModalProps> = React.memo(
               }}
               minRows={1}
             />
-           </div>
+
+           {buildForm && 
+           <>
+           <CustomInfo heading="Note" 
+           content="Allowing the addition of multiple pages in a single form will prevent you from using this form in a bundle later" />
+           <Form.Check
+             type="checkbox"
+             id="anonymouseCheckbox"
+             label={t("Allow adding multiple pages form in this form")}
+             checked={checked}
+             onChange={setChecked}
```

Comment:
can u pls add a data test id as it will be needd for testing @fahad-aot

Reply:
Added.